src/app/modules/analytics-config/components/edit-group/edit-group.component.ts

```diff
@@ -265,9 +271,11 @@ export class EditGroupComponent implements OnInit, OnDestroy {
 
   }
   public drop(event: CdkDragDrop<string[]>) {
+
     const previousIndex = parseInt(event.previousContainer.id.replace('widget-' + this.groupIndex + '-', ''), 10);
     const newIndex = parseInt(event.container.id.replace('widget-' + this.groupIndex + '-', ''), 10);
```

Comment:
I think that this part is not needed anymore

Reply:
fixed